src/components/LuxInputMultiselect.vue

```diff
@@ -1,128 +1,137 @@
 <template>
-  <div>
-    <label v-if="inputLabel" :for="id">{{ inputLabel }}</label>
-    <multiselect
-      v-model="vals"
-      :id="id"
-      :tag-placeholder="tagPlaceholder"
+  <div class="autocomplete-container">
+    <lux-autocomplete-input
+      :items="props.items"
+      @selected="addSelected($event)"
+      ref="autocomplete"
       :placeholder="placeholder"
-      label="name"
-      track-by="code"
-      :options="opts"
-      :multiple="true"
-      :taggable="true"
-      @tag="addTag"
-    ></multiselect>
-    <pre class="language-json"><code>{{ vals  }}</code></pre>
+    />
+    <lux-icon-base class="search-icon"><lux-icon-search></lux-icon-search></lux-icon-base>
   </div>
+  <span v-if="selectedItemsLabel" class="selected-item">{{ selectedItemsLabel }}</span>
+  <ul class="selected-items">
+    <li v-if="selectedItems.length == 0" class="selected-item">{{ noneSelectedLabel }}</li>
+    <li v-else v-for="item in selectedItems" :key="item" class="selected-item">
+      <!--
+        @slot item -- used to adjust the style and format of the items you have selected
+          @binding {object} itemProps an individual item that you would like to style
+      -->
+      <slot name="item" :itemProps="item">{{ item.label }}</slot>
+      <lux-input-button
+        @button-clicked="removeItem(item)"
+        type="button"
+        variation="icon"
+        icon="close"
+        size="small"
+        class="remove-item"
+      >
+      </lux-input-button>
+    </li>
+  </ul>
+  <!--
+  @slot hiddenInput -- useful for embedding this component into an HTML form
```

Comment:
THis is actually for passing all the values back to rails during the form submit.